Go 1.16 specific test error

On Fedora Rawhide with Go 1.16 beta 1, I get the following test error:

```
Testing    in: /builddir/build/BUILD/properties-1.8.4/_build/src
         PATH: /builddir/build/BUILD/properties-1.8.4/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/properties-1.8.4/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags " -X github.com/magiconair/properties/version=1.8.4 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld  '"
      testing: github.com/magiconair/properties
github.com/magiconair/properties
2021/01/16 17:04:51 properties: doesnotexist.properties not found. skipping
2021/01/16 17:04:51 properties: /tmp/properties477326312foo not found. skipping
2021/01/16 17:04:51 properties: /tmp/properties722766887foo not found. skipping
2021/01/16 17:04:51 properties: http://127.0.0.1:38403/c returned 404. skipping
	assert.go:62: got time: invalid duration "ghi" which does not match time: invalid duration ghi
--- FAIL: TestMustGetParsedDuration (0.00s)
FAIL
exit status 1
FAIL	github.com/magiconair/properties	0.025s
```

It seems the following code does not handle version over 1.15:

https://github.com/magiconair/properties/blob/0e5c09062c2daef666bd279e2d2d9f25f218f2be/properties_test.go#L562-L569

--- a/properties_test.go
+++ b/properties_test.go
@@ -560,12 +560,17 @@ func TestMustGetParsedDuration(t *testing.T) {
 	assert.Equal(t, p.MustGetParsedDuration("key"), 123*time.Millisecond)
 
 	ver := runtime.Version()
-	switch {
-	// gotip and go1.15 will return `time: invalid duration "ghi"`
-	case !strings.HasPrefix(ver, "go") || strings.HasPrefix(ver, "go1.15"):
-		assert.Panic(t, func() { p.MustGetParsedDuration("key2") }, `time: invalid duration "ghi"`)
-	default:
-		assert.Panic(t, func() { p.MustGetParsedDuration("key2") }, `time: invalid duration ghi`)
+	if strings.HasPrefix(ver, "go") {
+		ver_minor := ver[4:6]
+		switch {
+		// gotip and go1.15 will return `time: invalid duration "ghi"`
+		case ver_minor >= "15":
+			assert.Panic(t, func() { p.MustGetParsedDuration("key2") }, `time: invalid duration "ghi"`)
+		default:
+			assert.Panic(t, func() { p.MustGetParsedDuration("key2") }, `time: invalid duration ghi`)
+		}
+	} else {
+			assert.Panic(t, func() { p.MustGetParsedDuration("key2") }, `time: invalid duration "ghi"`)
 	}
 
 	assert.Panic(t, func() { p.MustGetParsedDuration("invalid") }, "unknown property: invalid")

